notification-channel-config-service-impl/src/test/java/org/hypertrace/notification/config/service/NotificationChannelConfigServiceRequestValidatorTest.java

```diff
@@ -81,12 +95,38 @@ public void testValidateWebhookHttpsSupport() {
         getNotificationChannelMutableData("http://localhost:9000/test");
     notificationChannelConfigServiceRequestValidator.validateWebhookHttpSupport(
         notificationChannelMutableDataWithHttpUrl, updatedNotificationChannelConfig);
+
+    // Update config with http support enabled and verify no exceptions for http url
+    notificationChannelConfigServiceRequestValidator.validateUpdateNotificationChannelRequest(
+        RequestContext.forTenantId("tenant1"),
+        UpdateNotificationChannelRequest.newBuilder()
+            .setId("id1")
+            .setNotificationChannelMutableData(notificationChannelMutableDataWithHttpUrl)
+            .build(),
+        updatedNotificationChannelConfig);
+  }
+
+  private static UpdateNotificationChannelRequest getUpdateNotificationChannelRequestWithHttpUrl() {
+    return UpdateNotificationChannelRequest.newBuilder()
+        .setNotificationChannelMutableData(
+            NotificationChannelMutableData.newBuilder()
+                .setChannelName("channel1")
+                .addWebhookChannelConfig(
+                    WebhookChannelConfig.newBuilder()
+                        .setUrl("http://localhost:9000/url")
```

Comment:
Should we parametrise the url for this method as was done for ` getNotificationChannelMutableData`?